lib/sasl/test/release_handler_SUITE.erl

```diff
@@ -20,7 +20,7 @@
 %% %CopyrightEnd%
 %%
 -module(release_handler_SUITE).
-
+ 
```

Comment:
✂️

Reply:
It is there to trigger the sasl tests on github actions :)